### PR TITLE
ref(tests): Convert Arithmetic input to rtl

### PR DESCRIPTION
**Pull request by priscilawebdev**
convert file to react testing library

### CODE REVIEW

File: static/app/views/eventsV2/table/arithmeticInput.tsx [hunk 1]
```diff
@@ -307,36 +307,39 @@ type TermDropdownProps = {
...
-                    <DropdownItemTitleWrapper>{option.value}</DropdownItemTitleWrapper>
-                  </DropdownListItem>
-                );
-              })}
-              {options.length === 0 && <Info>{t('No items found')}</Info>}
-            </Fragment>
-          );
-        })}
-      </DropdownItemsList>
+      {isOpen && (
```
Comment (evanpurkhiser) [inline static/app/views/eventsV2/table/arithmeticInput.tsx:310]: Why does it need this?

Comment (priscilawebdev) [reply static/app/views/eventsV2/table/arithmeticInput.tsx:310]: because otherwise everything gets rendered in the DOM  when it shouldn't

Comment (evanpurkhiser) [reply static/app/views/eventsV2/table/arithmeticInput.tsx:310]: Is it not hidden in some way?

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/views/eventsV2/table/arithmeticInput.tsx (hunk 1 above)
Comment (priscilawebdev) [reply static/app/views/eventsV2/table/arithmeticInput.tsx:310]: yes through CSS

Comment (evanpurkhiser) [reply static/app/views/eventsV2/table/arithmeticInput.tsx:310]: Like how was this working before? haha

Comment (evanpurkhiser) [reply static/app/views/eventsV2/table/arithmeticInput.tsx:310]: Ahhh hmm. Should we remove the CSS then?

Comment (priscilawebdev) [reply static/app/views/eventsV2/table/arithmeticInput.tsx:310]: https://github.com/getsentry/sentry/pull/41007#discussion_r1013905715

Comment (priscilawebdev) [reply static/app/views/eventsV2/table/arithmeticInput.tsx:311]: we could, but there is a border and I was afraid that I could cause some jumping effect... maybe I can better take a look into it in a follow up

**Pull request merged**